Installers/serverConfig.sh

```
@@ -0,0 +1,290 @@
+
+    if [ $exit_code -ne 0 ]; then
+        print_warn "Script exited with code: $exit_code"
+    fi
+
+    read -rp "Press [Enter] to return to menu or type 'exit': " next_action
+    if [ "$next_action" = "exit" ]; then
+        echo -e "\n${GREEN}Goodbye!${NC}"
+        exit 0
+    fi
+}
+}
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)